app/workers/generate_test_applications.rb

```diff
@@ -4,20 +4,20 @@ class GenerateTestApplications
   def perform
     raise 'You can\'t generate test data in production' if HostingEnvironment.production?
 
-    TestApplications.create_application states: [:unsubmitted]
-    TestApplications.create_application states: [:awaiting_references]
-    TestApplications.create_application states: [:application_complete]
-    TestApplications.create_application states: [:awaiting_provider_decision] * 3
-    TestApplications.create_application states: [:offer] * 2
-    TestApplications.create_application states: %i[offer rejected]
-    TestApplications.create_application states: [:rejected] * 2
-    TestApplications.create_application states: [:offer_withdrawn]
-    TestApplications.create_application states: [:declined]
-    TestApplications.create_application states: [:accepted]
-    TestApplications.create_application states: [:accepted_no_conditions]
-    TestApplications.create_application states: [:recruited]
-    TestApplications.create_application states: [:conditions_not_met]
-    TestApplications.create_application states: [:enrolled]
-    TestApplications.create_application states: [:withdrawn]
+    TestApplications.new.create_application states: [:unsubmitted]
```

Comment:
Would it work to reuse the same instance of `TestApplications` instead of `new`ing up a fresh one each line?